scrapers/GirlsOutWest.yml

```diff
@@ -4,20 +4,23 @@ sceneByURL:
     url:
       - tour.girlsoutwest.com/
     scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - tour.girlsoutwest.com/models/
+    scraper: performerScraper
 xPathScrapers:
   sceneScraper:
     scene:
       Title:
         selector: //div[@class='trailerTitle']/div/h3/text()
-        postProcess:
```

Comment:
Can you comment out instead of deleting the pp? 
Some users might not want the  extra perfromers part of the title.